nexuse2e-webapp/src/main/frontend/src/app/data/data.service.ts

```diff
@@ -16,12 +19,47 @@ export class DataService {
     }
   }
 
-  get<T>(path: string): Promise<T> {
+  private get<T>(path: string, cache: boolean): Promise<T> {
```

Comment:
Is a default value for cache and removing all true arguments making the code a little better to understand?
```suggestion
  private get<T>(path: string, cache = true): Promise<T> {
```

Reply:
Thank you, we were actually looking for this syntax ;)